src/test/regress/pg_regress.c

```diff
@@ -91,8 +91,8 @@ static int	port = -1;
 static char *user = NULL;
 static char *srcdir = NULL;
 static _stringlist *extraroles = NULL;
-char *initfile = NULL;
-char *expected_statuses_file = NULL;
+static char *initfile = "./init_file";
```

Comment:
Should be a error from git, another irrelevant commit is involved. This is why I close it at once.